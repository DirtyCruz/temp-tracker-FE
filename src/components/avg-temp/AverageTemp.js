import React from 'react';
import PropTypes from 'prop-types';

function AverageTemp({ avgTemp }) {
  return (
    <>
      <h3>Average Temperatures</h3>
      <h4>{avgTemp}</h4> 
    </>
  );
}

AverageTemp.propTypes = {
  avgTemp: PropTypes.number.isRequired
};

export default AverageTemp;


