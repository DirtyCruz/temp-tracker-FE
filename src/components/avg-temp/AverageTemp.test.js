import React from 'react';
import { shallow } from 'enzyme';
import AverageTemp from './AverageTemp';

describe('AverageTemp component', () => {
  it('renders AverageTemp', () => {
    const wrapper = shallow(<AverageTemp avgTemp={55}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
