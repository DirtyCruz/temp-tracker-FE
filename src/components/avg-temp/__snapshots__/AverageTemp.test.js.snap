// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AverageTemp component renders AverageTemp 1`] = `
<Fragment>
  <h3>
    Average Temperatures
  </h3>
  <h4>
    55
  </h4>
</Fragment>
`;
